cmd/karavictl/cmd/role_create.go

```diff
@@ -47,6 +47,9 @@ var roleCreateCmd = &cobra.Command{
 		var rff roles.JSON
 		for _, v := range roleFlags {
 			t := strings.Split(v, "=")
+			if len(t) < 5 {
```

Comment:
Could use a const here for the value of 5 to improve code readability

Reply:
fixed